src/index.js

```diff
@@ -1 +0,0 @@
-export default from './FlexView';
```

Comment:
why did you remove this? this is definitely a big breaking change. is it necessary?